extern/storage-sealing/states_sealing.go

```diff
@@ -360,7 +360,7 @@ func (m *Sealing) handleSubmitPreCommitBatch(ctx statemachine.Context, sector Se
 
 	params, deposit, _, err := m.preCommitParams(ctx, sector)
 	if params == nil || err != nil {
-		return ctx.Send(SectorChainPreCommitFailed{xerrors.Errorf("preCommitParams: %w", err)})
+		return err
```

Comment:
I think it's more correct to still send the event if we see the error
```suggestion
		if err != nil {
			return ctx.Send(SectorChainPreCommitFailed{xerrors.Errorf("preCommitParams: %w", err)})
		}
		return nil // event was sent in preCommitParams
```